internal/utils/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const DEVNET_NETWORK_URL = 'https://devnet.fuel.network/v1/graphql';
+export const TESTNET_NETWORK_URL = 'https://testnet.fuel.network/v1/graphql';
```

Comment:
@maschad @Dhaiwat10  Not gonna lie; a new package for this felt a bit over in the end. 🫠

Reply:
I agree @arboleya if it remains only for these two constants but if we eventually migrate any other utilities that we want to internalize here then it may be more worthwhile. I don't have a list of those current utilities atm though 😅

Reply:
Yah +1, we could've left the constants internally in `account` but not exported them from `fuels` (once deprecated) 🤔

Reply:
Gonna merge this PR with this new package, I think we'll probably be adding other constants and utilities to this package in the future like @maschad mentioned